src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,9 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery game = new Lottery();
+        for (int i = 0; i < 3; i++) {
+            System.out.println(game);
```

Comment:
```suggestion
            System.out.println(game.getRandomBall());
```
have you tested your program? :) We need to print the result of the getRandomBall method call, not just lottery, which doesn't even have toString overrided to be human readable